From c5e757a198321208c6b1e99313652577c263ecc0 Mon Sep 17 00:00:00 2001
From: Fushan Wen <qydwhotmail@gmail.com>
Date: Fri, 7 Apr 2023 22:02:11 +0800
Subject: [PATCH] gtk3: restore old icon size for titlebutton

`background-size` results in different sizes on X11 and Wayland.

BUG: 468203
FIXED-IN: 5.27.5


(cherry picked from commit bfe2ce620d395c943bb13a646295948b5bffbcc3)
---
 src/gtk3/widgets/_window_decorations.scss | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/gtk3/widgets/_window_decorations.scss b/src/gtk3/widgets/_window_decorations.scss
index aef28cac..6568e6bf 100644
--- a/src/gtk3/widgets/_window_decorations.scss
+++ b/src/gtk3/widgets/_window_decorations.scss
@@ -74,11 +74,11 @@ window {
 	.titlebar {
 		&.default-decoration button.titlebutton {
 			padding: 1px;
-			min-height: 24px;
-			min-width: 24px;
+			min-height: 18px;
+			min-width: 18px;
 			margin: 0;
 		}
-		$titlebutton-size: 24px 24px;
+		$titlebutton-size: 18px 18px;
 		button.titlebutton {
 			&:hover,
 			&:active,
-- 
GitLab

